### PR TITLE
#process proto changes

### PR DESCRIPTION
**Pull request by Chemssou**

### CODE REVIEW

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pporder/PPOrderLinesViewFactory.java [hunk 1]
```diff
@@ -155,7 +155,8 @@ private ViewLayout createViewLayout(final WindowId windowId)
 	{
 		return ImmutableList.of(
 				createProcessDescriptorForIssueReceiptWindow(de.metas.ui.web.pporder.process.WEBUI_PP_Order_Receipt.class),
-				createProcessDescriptorForIssueReceiptWindow(de.metas.ui.web.handlingunits.process.WEBUI_M_HU_Pick.class),
```
Comment (metas-ts) [inline backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pporder/PPOrderLinesViewFactory.java:158]: what is `WEBUI_M_HU_Pick` about? is it now obsolete? if yes, please remove it altogether

Comment (metas-ts) [reply backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pporder/PPOrderLinesViewFactory.java:158]: ok, so I guess `WEBUI_M_HU_Pick` will stay in its own right, jsut won't be accessible from the production modal, right?

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pporder/process/WEBUI_PP_Order_Pick_HU.java [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+		// NOTE: we are not moving the HU to shipment schedule's locator.
+		pickingCandidateService.processForHUIds(ImmutableSet.of(huId),
+												shipmentScheduleId,
+												OnOverDelivery.ofTakeWholeHUFlag(isTakeWholeHU),
+												getView().getPpOrderId());
+	}
+
+	private Set<HuId> retrieveSelectedHuIds()
+	{
+		final DocumentIdsSelection selectedRowIds = getSelectedRowIds();
```
Comment (metas-ts) [inline backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pporder/process/WEBUI_PP_Order_Pick_HU.java:192]: please extract the code that is common with WEBUI_PP_Order_PrintLabel into a super-class, utility-class or elsewhere, to deduplicate it

Comment (metas-ts) [reply backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pporder/process/WEBUI_PP_Order_Pick_HU.java:192]: PS: also please look into doing the same for the code that is common with WEBUI_M_HU_Pick

Comment (Chemssou) [reply backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pporder/process/WEBUI_PP_Order_Pick_HU.java:192]: >> ok, so I guess WEBUI_M_HU_Pick will stay in its own right, jsut won't be accessible from the production modal, right?
Yes indeed

**Pull request closed without merging**